types/siren-parser/index.d.ts

```diff
@@ -0,0 +1,143 @@
+// Type definitions for siren-parser 8.2
+// Project: https://github.com/Brightspace/node-siren-parser
+// Definitions by: Dillon Redding <https://github.com/dillonredding>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.7
```

Comment:
```suggestion
// TypeScript Version: 3.0
```
this should work ok, as the `superagent` states this version, this seems to be only package-depending, not depending on the TS language features, correct?

Reply:
I added that per a recommendation from the linter:

```
$ npm run lint siren-parser

> definitely-typed@0.0.3 lint C:\dev\DefinitelyTyped
> dtslint types "siren-parser"

Error: C:/dev/DefinitelyTyped/types/siren-parser/siren-parser-tests.ts:103:6     
ERROR: 103:6   expect  Compile error in typescript@3.6 but not in typescript@3.7.
Fix with a comment '// Minimum TypeScript Version: 3.7' just under the header.   
Expression expected.
ERROR: 103:10  expect  Compile error in typescript@3.6 but not in typescript@3.7.
Fix with a comment '// Minimum TypeScript Version: 3.7' just under the header.   
':' expected.
...
```

Seems to be where I'm doing `variable?.prop;`. To be honest, I'm just getting back into TypeScript after a few years. Is there a better approach here?

Reply:
I see, but the problem is only in the tests (Elvis operator). I usually opt to older syntax in test, to not limit the scope of the usage of the type definition itself.

Reply:
Gotcha. I replaced the `?.`s with `if` checks and lowered the minimum version.